src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -31,7 +31,6 @@ public LiveState() {
     nameNodeHosts = new HashSet<>();
     stagingTasks = new HashSet<>();
   }
-
```

Comment:
Can you please add this blank line back in? It's nice to separate functions from each other with whitespace.